testing/btest/bifs/to_count.zeek

```diff
@@ -19,6 +19,9 @@ event zeek_init()
 	print to_count("7");
 	print to_count("");
 	print to_count("-5");
+	print to_int("10101100", 2);
+	print to_int("43", 8);
+	print to_int("C3", 16);
```

Comment:
Should these be tests for `to_count`? Maybe add a test for the `0xFF` form of hex numbers as well.

Reply:
Yes, that was a copy/pasta, thanks for catching. And having more tests for the "0x" format sounds good, I added one to each.